src/wp-includes/class-wp-block-supports.php

```diff
@@ -0,0 +1,196 @@
+<?php
```

Comment:
Let's move this file to `src/wp-includes/block-supports/class-wp-block-supports.php`.

Reply:
I actually reverted this change, I believe it's better there with the other classes and `src/wp-includes/block-supports` only includes the actual block supports not the framework for it.